test/test/publisher_subscriber/PublisherSubscriberInteraction.cpp

```diff
@@ -173,6 +173,17 @@ TEST_P(PublisherSubscriberNoLost, PubSub1ContinousFragmentedTopic)
     publisher_.publish(1, 0x80, 1, message);
 }
 
+TEST_P(PublisherSubscriberNoLost, PubSub1WithPing)
+{
+    size_t message_number = 1;
+    publisher_.publish(1, 0x80, message_number, SMALL_MESSAGE);
+    subscriber_.request_data(1, 0x80, SMALL_MESSAGE);
+    subscriber_.ping_agent(transport_);
```

Comment:
Should we ping multiple times to get a topic?

Reply:
It depends on when the pong arrives. Maybe if pong arrives before the topic, the topic is not processed